### DISCUSSION TITLE
[Feature Request] Expand and organize FlyonUI Theme Options

### DISCUSSION THREAD

**Discussion by GrungeElFz** [themeselection/flyonui#27] [Ideas / Request new feature]
## Feature Request: Expand and organize FlyonUI Theme Options

Currently, FlyonUI offers a limited selection of built-in themes:

- `Light`
- `Dark`
- `Gourmet`
- `Corporate`
- `Luxury`
- `Soft`

Many popular UI libraries offer a diverse range of built-in themes, providing developers and users accessibility for a quick test for a head start. Expanding the theme library would help FlyonUI's attracting a wider developer/user base. 

### Proposal:

1. **Port Popular Themes:**
   Introduce a wider variety of themes from well-established UI libraries:

   *   **daisyUI**: 20+ more themes to port.
   *   **Vercel**: Well-known distinctive color scheme.
   *   **Mintlify**:  Minimalist and clean theme.
   *   **Tailwind**: Explore popular community themes built with Tailwind.

2. **Improve Theme Organization:**
   Refactor the theme list to be alphabetically ordered, with the exception of keeping `Light` and `Dark` as the first two options for easy access. This will improve navigation and discoverability as the theme library grows.

**Benefits:**

*   **Enhanced Developer/User Experience:** Offer more built-in options to for a quicker development and test process.
*   **Increased Adoption:** Make FlyonUI more appealing to a broader audience by offering a wider range of design options from the start.
*   **Improved Accessibility:** Include themes specifically designed for accessibility, such as `Night` and `Black` from daisyUI.

**Contribution:**

I'd be happy to contribute to this effort and help with the following:

*   **Code Implementation:** Assist in porting themes from daisyUI, Vercel, and etc.
*   **Testing:** Thoroughly test any newly added themes to ensure they integrate seamlessly with FlyonUI's components and features.
*   **Documentation:**  Help document the new themes, including their visual styles and any specific theme information.

<br/>

Thank you for considering these improvements ! 🙏

**Comment by yagnikvamja** [comment]
Hello @GrungeElFz,

Thank you for your thoughtful suggestion! We truly appreciate your input.

We’re pleased to share that we plan to introduce additional themes in future updates. If you have any specific ideas for a contrast theme, please feel free to suggest them. We’d be happy to review your proposal, and if it aligns with our vision, we’ll gladly incorporate it into FlyonUI.

Your contributions are always valued, and we look forward to hearing from you.

~Best regards,

**Reply by FordSmh** [reply]
Would be nice to have default preline notion-like theme as an option or maybe someone can prepare color set and post here
